apps/services/auth/ids-api/src/app/login-restrictions/login-restrictions.controller.spec.ts

```diff
@@ -90,29 +90,81 @@ describe('LoginRestrictionsController', () => {
         })
       })
 
-      it('should return 400 Bad Request for empty phone number', async () => {
+      it.each`
+        phoneNumber
+        ${''}
+        ${'1234567'}
+        ${'12345678901234567890'}
+        ${'123abcd'}
+        ${'+4543218765'}
+      `(
+        'should return 400 Bad Request for invalid phone number $phoneNumber',
+        async (phoneNumber) => {
+          // Arrange
+          await loginRestrictionModel.create({
+            nationalId: '1234567890',
+            phoneNumber: userPhoneNumber.replace('-', ''),
+            until: startOfDay(addDays(new Date(), 8)),
+          })
+
+          // Act
+          const response = await server
+            .get(path)
+            .set('X-Param-Phone-Number', phoneNumber)
```

Comment:
X-Query-...?

Reply:
These are refactored old tests on the `login-restrictions/.phone-number` endpoints. 
Will be removed when I remove the endpoint.